versions/v0.7.0/values.yaml

```diff
@@ -29,6 +29,12 @@ koordlet:
   log:
     # log level for koordlet
     level: "4"
+  volume:
```

Comment:
volume => hostDirs
please define variables by function instead of location, e.g. kubeletConfigDir, kubeletLibDir, koordProxyRegisterDir, koordletSockDir